js/murdock.js

```diff
@@ -126,7 +128,7 @@ function pr_status(status) {
   }
   if (status.failed_jobs) {
     if (status.failed_jobs.length > 0) {
-      failed_jobs_html += "<h5><strong>Failed jobs:</strong></h5>";
+      failed_jobs_html += "<strong>Failed jobs:</strong>";
```

Comment:
Why not keep this a heading, at least for semantic purposes?

Reply:
Because the text size is too big:

**this PR as-is**:

![image](https://user-images.githubusercontent.com/1375137/122584394-9a57a000-d05a-11eb-9ba4-304e17d83def.png)


**this PR with a h5 heading**:

![image](https://user-images.githubusercontent.com/1375137/122584315-814eef00-d05a-11eb-83f7-fbc29a2acd41.png)


I prefer the first version.

Reply:
And when you go up to `h6`?

Reply:
I tried it initially but I didn't find it worth:

![image](https://user-images.githubusercontent.com/1375137/122678469-5e057a80-d1e7-11eb-8440-5a6337612c8c.png)

Reply:
Playing a bit with it and I'll push an update that uses h6 and adds some vertical margins.